anchors/bootstrapper.go

```diff
@@ -31,8 +32,12 @@ func (Bootstrapper) Bootstrap(ctx map[string]interface{}) error {
 		return err
 	}
 
-	var repo AnchorRepository
-	repo = NewEthereumAnchorRepository(cfg, repositoryContract, ethereum.GetClient)
+	if _, ok := ctx[bootstrap.BootstrappedQueueServer]; !ok {
```

Comment:
can you move the `BootstrappedQueueServer` to queue package instead. This is would help us remove the cyclic dependencies

Reply:
thats the reason why the constants were created in the `bootstrap` package. it contains only interfaces, if someone creates cyclic dependancies using this there is something wrong with the code.